source/val/validate_decorations.cpp

```diff
@@ -779,6 +780,19 @@ spv_result_t CheckDecorationsOfEntryPoints(ValidationState_t& vstate) {
         }
         const spv::StorageClass storage_class =
             var_instr->GetOperandAs<spv::StorageClass>(2);
+        if (vstate.version() >= SPV_SPIRV_VERSION_WORD(1, 5)) {
```

Comment:
Why is this version gated on 1.5? SPV_EXT_mesh_shader requires SPIR-V 1.4. What is intended to be checked here?

Reply:
It should be 1.4, I am not sure why i added 1.5.
The intention here is to satisfy the requirement in SPV_EXT_mesh_shader spec
"There can be at most one <id> of type OpVariable with storage class TaskPayloadWorkgroupEXT associated with an OpEntryPoint. This OpVariable should be a global OpVariable." 

Basically we cannot have more then one taskPayloadSharedEXT type global variable associated with one entrypoint